src/react-integration/hooks/useFetcher.ts

```diff
@@ -17,7 +17,11 @@ export default function useFetcher<
   Params extends Readonly<object>,
   Body extends Readonly<object> | void,
   S extends Schema
->(fetchShape: FetchShape<S, Params, Body>, throttle = false) {
+>(
+  fetchShape: FetchShape<S, Params, Body>,
+  throttle = false,
+  onlyIfStale = false,
```

Comment:
not really in love with this variable name, doesn't really say _what_ is happening only if stale.